tests/testthat/test-log_formatter.R

```diff
@@ -1,46 +1,46 @@
 testthat::test_that("teal.logger formats NULL asis", {
   out <- logger::log_info("null: {NULL}")
-  testthat::expect_identical(out$default$message, "null: NULL")
+  testthat::expect_equal(out$default$message, "null: NULL")
```

Comment:
To add more context:
<img src="https://github.com/user-attachments/assets/8b9fb516-d6f3-4938-af12-e30dd46ccf52" width="250" />